apps/src/applab/sanitizeHtml.js

```diff
@@ -74,6 +80,9 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
           attribs: attribs
         };
       }
+    },
+    exclusiveFilter: function (element) {
+      return (ignoredTags.indexOf(element.tag) !== -1);
```

Comment:
It looks like `element.tag` is always lower case, based on a quick test. So are attributes.